first_lessons_dushanbe/3x3.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+if __name__ == '__main__':
+    a = []
```

Comment:
nit: you can use a short hand for two dimensional array initialization : `[ [0]*3 for i in range(3)]`

Reply:
)))